ngraph/test/runtime/dynamic/dynamic_backend.cpp

```diff
@@ -387,7 +381,7 @@ void runtime::dynamic::DynamicTensor::release_storage()
 }
 
 void runtime::dynamic::DynamicTensor::make_storage(const element::Type& element_type,
```

Comment:
Probably it is worth to investigate if DynamicTensor is still needed since `m_wrapped_backend->create_dynamic_tensor` was added. However, it is rather topic for separate PR.